maintainer/conda/environment.yml

```
@@ -33,3 +33,4 @@ dependencies:
       - msmb_theme==1.2.0
       - sphinx-sitemap==1.0.2
       - packaging
+      - pyedr
```

Same here query versioning?

---

See above

---

```suggestion
      - pyedr>=0.7.0
```
I'm not sure I can find what this relates to, but we probably want to set this to >=0.7.0 too?